integration-tests/docker/environment-configs/coordinator

```diff
@@ -32,7 +32,11 @@ druid_coordinator_startDelay=PT5S
 druid_manager_lookups_hostUpdateTimeout=PT30S
 druid_manager_lookups_period=10000
 druid_manager_lookups_threadPoolSize=2
+druid_manager_config_pollDuration=PT10S
+druid_manager_rules_pollDuration=PT10S
+druid_manager_segments_pollDuration=PT2S
 druid_auth_basic_common_cacheDirectory=/tmp/authCache/coordinator
 druid_auth_unsecuredPaths=["/druid/coordinator/v1/loadqueue"]
 druid_server_https_crlPath=/tls/revocations.crl
-druid_coordinator_period_indexingPeriod=PT180000S
\ No newline at end of file
+druid_coordinator_period_indexingPeriod=PT180000S
+druid_coordinator_period=PT1S
```

Comment:
Is this too aggressive? :sweat_smile:

Reply:
Ha ha 🙂

I think it's OK — there are going to be very few segments in the integration tests, so a coordinator run should be able to finish very quickly.